dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/export/enrollment/EnrollmentServiceTest.java

```diff
@@ -793,6 +792,55 @@ void shouldFailWhenRequestingListOfEnrollmentsAndAtLeastOneNotAccessible() {
         exception.getMessage());
   }
 
+  @Test
+  void testGetEnrollmentById() {
+    manager.save(enrollmentA);
+    manager.save(enrollmentB);
+    assertEquals(enrollmentA, manager.get(Enrollment.class, enrollmentA.getUid()));
+    assertEquals(enrollmentB, manager.get(Enrollment.class, enrollmentB.getUid()));
+  }
+
+  @Test
+  void testGetEnrollmentsByTrackedEntityProgramAndEnrollmentStatus()
+      throws ForbiddenException, BadRequestException {
+    manager.save(enrollmentA);
+    Enrollment enrollment1 = createEnrollment(programA, trackedEntityA, orgUnitA);
+    enrollment1.setStatus(EnrollmentStatus.COMPLETED);
+    manager.save(enrollment1, false);
+    Enrollment enrollment2 = createEnrollment(programA, trackedEntityA, orgUnitA);
+    enrollment2.setStatus(EnrollmentStatus.COMPLETED);
+    manager.save(enrollment2, false);
+
+    List<Enrollment> enrollments =
+        enrollmentService.getEnrollments(trackedEntityA, programA, EnrollmentStatus.COMPLETED);
+    assertContainsOnly(List.of(enrollment1, enrollment2), enrollments);
```

Comment:
Please arrange tests in setup, code under test and assertions separated by a newline. This makes it easier to read.

Also this test should be spilt in two as they can fail independently.